modules/base/rendering/renderablemodel.cpp

```diff
@@ -136,110 +142,111 @@ namespace {
         "Enable Opacity Blending",
         "Enable Opacity Blending."
     };
+
+    struct [[codegen::Dictionary(RenderableModel)]] Parameters {
+        // The file or files that should be loaded in this RenderableModel. The file can
+        // contain filesystem tokens or can be specified relatively to the
+        // location of the .mod file.
+        // This specifies the model that is rendered by the Renderable.
+        std::filesystem::path geometryFile;
+
+        enum class ScaleUnit {
+            Nanometer,
+            Micrometer,
+            Millimeter,
+            Centimeter,
+            Decimeter,
+            Meter,
+            Kilometer
+        };
+
+        // The scale of the model. For example if the model is in centimeters
+        // then ModelScale = Centimeter or ModelScale = 0.01
+        std::optional<std::variant<ScaleUnit, float>> modelScale;
+
+        // Set if invisible parts (parts with no textures or materials) of the model
+        // should be forced to render or not.
+        std::optional<bool> forceRenderInvisible;
+
+        // [[codegen::verbatim(enableAnimationInfo.description)]]
+        std::optional<bool> enableAnimation;
+
+        // The date and time that the model animation should start.
+        // In format 'YYYY MM DD hh:mm:ss'.
+        std::optional<std::string> animationStartTime [[codegen::dateTime()]];
```

Comment:
Currently, all of the attributes (with the exception of the `Dictionary` are all lowercaps.  I think it would be good to have that here as well and use `codegen::datetime` instead for consistency